index.js

```diff
@@ -46,24 +47,29 @@ const getTotalCharged = (scrapeResult) => {
       totalCharged -= txn.chargedAmount;
     });
   });
-  return totalCharged;
-};
-
-const sendTelegramMessage = (message) => {
-  const bot = new TelegramBot(process.env.TELEGRAM_BOT_TOKEN);
-  const chatId = process.env.TELEGRAM_CHAT_ID;
-  bot.sendMessage(chatId, message);
+  return Math.round(totalCharged * 100) / 100;
```

Comment:
There's no need for that if you've already done the rounding afterwards.

Reply:
This is the rounding for the price, the other is for the `exceed_percent`